connectivity/manifests/allow-all-except-world-pre-v1.11.yaml

```diff
@@ -13,16 +13,14 @@ spec:
     - health
   - toEndpoints:
     - {}
-  # When node-local-dns is deployed with local IP,
-  # Cilium labels its ip as world.
-  # This change prevents failing the connectivity
-  # test for such environments.
+  {{- if .DNSCIDR }}
```

Comment:
I'm not sure we should make this dependent on .DNSCIDR being set. Shouldn't we keep the existing policy in case the user didn't specify .DNSCIDR/--dns-cidr?

Reply:
I don't know, but https://github.com/cilium/cilium-cli/pull/997 is not released (yet), so this is not a regression.